nicos_ess/loki/gui/script_generator.py

```diff
@@ -138,16 +138,15 @@ def generate_script(self, labeled_data, trans_duration_type,
 class ScriptGenerator:
     @classmethod
     def from_trans_order(cls, trans_order):
-        if trans_order == TransOrder.TRANSFIRST:
-            return TransFirst()
-        elif trans_order == TransOrder.SANSFIRST:
-            return SansFirst()
-        elif trans_order == TransOrder.TRANSTHENSANS:
-            return TransThenSans()
-        elif trans_order == TransOrder.SANSTHENTRANS:
-            return SansThenTrans()
-        elif trans_order == TransOrder.SIMULTANEOUS:
-            return Simultaneous()
+        order_attributes = {
```

Comment:
```suggestion
        classes_by_trans_order = {
```
Might as well make it completely obvious what it is.
Don't forget to change it in the code below ;)